Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2511,11 +2511,21 @@ extension BrowserViewController: WKUIDelegate {
             return UIMenu(title: url.absoluteString, children: actions)
         }
         
-        let config = UIContextMenuConfiguration(identifier: nil, previewProvider: nil, actionProvider: actionProvider)
+        let linkPreview: UIContextMenuContentPreviewProvider = {
+            return SFSafariViewController(url: url)
```

Comment:
Is it possible to make our own preview provider that uses BVC in some way (so shields are loaded in and such)